### ISSUE TITLE
Travis Brown abuse of public spaces

### ISSUE THREAD

**Issue by hmemcpy** [typelevel/general#112]
Shortly before deleting my twitter account this morning, I found myself mentioned *by my full name* by Travis Brown. This was in response to Wiem Zine El Abidine, a woman who shortly before tweeted that she was **afraid for her safety** walking the streets of Berlin
![image](https://user-images.githubusercontent.com/601206/141153855-1456ed50-1d79-4125-b7fb-914803d510fa.png)

Wiem is a Muslim woman, originally from Tunisia, whose only *crime* was that she came into Scala via John de Goes, who mentored her and co-wrote the early version of ZIO with her, suffered an incredible amount of abuse from the supposed "community", so much so that she left twitter and subsequently left Scala altogether. Well done!

This morning, as she asked Travis to [reconsider his actions](https://twitter.com/wiemzin/status/1458128301684314112), and unblock her to continue the dialog, he responded with bunch of tweets shifting the blame back at her, concluding with this:
![image](https://user-images.githubusercontent.com/601206/141155895-3e621379-143b-4ffd-aee9-edf4dfc02497.png)

I don't know in what world this is considered normal behavior towards a minority member to gaslight her and accuse her of lying and imagining things, but for me it has certainly crossed the line.

I don't expect anything to be done here by Typelevel. As you claim, Travis Brown is not a Typelevel member, his projects are not part of this organization. Yet his behavior and actions are lauded and supported by nearly all of you. Why isn't the Scala Code of Conduct, by which you all abide, apply to him?

If this does not prevent his participation in your spaces, or warrant a public reprimand, then you have no moral compass to lecture others.

I'm done, feel free to close this issue.

**Comment by rossabaker** [comment]
Thank you for the report. Typelevel leadership is actively discussing its mission and role in the wider community.  We will take this under advisement, and issue a statement in coming days
